weblate/memory/forms.py

```diff
@@ -17,19 +18,9 @@ class UploadForm(forms.Form):
 
     file = forms.FileField(
         label=gettext_lazy("File"),
-        validators=[
-            FileExtensionValidator(
-                allowed_extensions=[
-                    "json",
-                    "tmx",
-                    "xliff",
-                    "po",
-                    "csv",
-                ]
-            )
-        ],
+        validators=[FileExtensionValidator(allowed_extensions=SUPPORTED_FORMATS)],
         help_text=gettext_lazy(
-            "You can upload a file of following formats: TMX, JSON, XLIFF, PO, CSV."
+            f"You can upload a file of following formats: {', '.join(SUPPORTED_FORMATS)}."
```

Comment:
This won't work well for localizing, you need to use format string instead to inject value into localized string returned by gettext.



```suggestion
            "You can upload a file of following formats: %s."
```

And then apend `% ', '.join(SUPPORTED_FORMATS)`.